console-extensions.json

```diff
@@ -7,14 +7,14 @@
       "component": { "$codeRef": "AboutPage" }
     }
   },
-  {
-    "type": "console.page/route",
-    "properties": {
-      "exact": true,
-      "path": "/cryostat/settings",
-      "component": { "$codeRef": "SettingsPage" }
-    }
-  },
+  // {
+  //   "type": "console.page/route",
+  //   "properties": {
+  //     "exact": true,
+  //     "path": "/cryostat/settings",
+  //     "component": { "$codeRef": "SettingsPage" }
+  //   }
+  // },
```

Comment:
Comments aren't technically legal JSON... I guess whatever consumes this is tolerant of them?

Reply:
Ah maybe it's my IDE being nice to me, I'll remove them